src/noteburst/worker/functions/nbexec.py

```diff
@@ -125,11 +89,36 @@ async def nbexec(
                 )
                 await slack_client.post(message)
```

Comment:
Are we losing the `Job ID` and `Attempt` fields in this non-authentication error case?

Reply:
Good catch.  I think we are.

Reply:
Uh, wait, now I'm really confused, because...no, we're not?

Reply:
Ahh crap sorry, yep, those were folded.  Looks good!